src/main.py

```diff
@@ -6,6 +6,7 @@
 import os
 
 import chords_simplification
+from recorded_file_fixing import convert_file
```

Comment:
imports should be organised from the longest to the shortest (or reversed), so move it one line up

Reply:
Done